apps/src/storage/levelbuilder/ManifestEditor.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+
+class ManifestEditor extends React.Component {
+  static propTypes = {
+    // Provided via Redux
+    libraryManifest: PropTypes.object.isRequired
+  };
+
+  render() {
+    return (
+      <div>
+        <h1>Edit Dataset Manifest </h1>
+        <textarea
+          id="content"
+          ref="content"
+          value={JSON.stringify(this.props.libraryManifest, null, 2)}
```

Comment:
Add a comment for the primitive types here

Reply:
Fixed